### DISCUSSION TITLE
是否支持阿里云的polardb？

### DISCUSSION THREAD

**Discussion by lx308033262** [hhyo/Archery#1931] [Q&A]
### 重现步骤

有没有在polardb上面使用archery的

### 预期外的结果

可以使用archery审核polardb

### 版本

v1.8.5

### 部署方式

Docker

**Comment by nick2wang** [comment]
ploardb据说是高度兼容mysql语法的，有人试过可用#1271，可以先当做mysql来配置实例，使用中如果有出现兼容性问题欢迎反馈，一起改进

**Comment by bluerfish** [comment]
polardb就是Mysql，我们的实例全部都是polardb，目前看起来使用没有问题，（慢日志这块不行）两个版本1.8.5和1.9.1都有使用。

**Reply by opsdba** [reply]
这个怎么做到支持polardb for mysql，目前我们polardb 多，而rds for mysql较少

**Reply by lx308033262** [reply]
> 这个怎么做到支持polardb for mysql，目前我们polardb 多，而rds for mysql较少

直接用就行，我们现在直接用的，没有出现什么状况

**Reply by opsdba** [reply]
我说的是慢日志，而不是对接实例
